client/helpers.py

```diff
@@ -0,0 +1,27 @@
+
+def parse(arg):

```

Comment:
This is not used anywhere, what will it be fore? Does not contribute to functionality yet.